src/apps/mesoscale-explorer/ui/states.tsx

```diff
@@ -362,3 +364,237 @@ export class SnapshotControls extends PluginUIComponent<{}> {
+                                }
+                            },
+                            radius: 5,
+                            bias: 1.0,
+                            blurKernelSize: 11,
+                            blurDepthBias: 0.5,
+                            resolutionScale: 1,
+                            color: Color(0x000000),
+                        }
+                    },
+                    shadow: {
+                        name: 'on',
+                        params: {
+                            bias: 0.4,
+                            maxDistance: 256,
+                            steps: 64,
+                            tolerance: 1.0,
+                        }
+                    },
+                    outline: { name: 'off', params: {} },
+                    gamma: 1.0,
+                    dof: {
+                        name: 'on',
+                        params: {
+                            blurSize: 9,
+                            blurSpread: 1.0,
+                            inFocus: 0.0,
+                            PPM: 200.0,
+                            center: 'camera-target',
+                            mode: 'sphere',
+                        }
+                    }
+                }
+            });
+        }
+        // ignore Light
+        const options = { ignoreLight: false, materialStyle: { metalness: 0, roughness: 0.2, bumpiness: 0 } };
+        this.update(options);
+    }
+
+    update = (options: any) => {
+        this.plugin.managers.structure.component.setOptions(options as StructureComponentManager.Options);
+        // this will make sure its applied to all the entities
+        // clipObject should get updated as well
+        updateReprParams(this.plugin, options);
+    };
+
+
+    render() {
+        const style_params = {
+            ignoreLight: StructureComponentManager.OptionsParams.ignoreLight,
+            celShaded: StructureComponentManager.OptionsParams.celShaded,
+            materialStyle: StructureComponentManager.OptionsParams.materialStyle,
+        };
+        return <>
+            <div className='msp-flex-row'>
+                <Button noOverflow title='Applies default representation preset and sets outline and occlusion effects to default' onClick={() => this.default()} style={{ width: 'auto' }}>
+                    Default
+                </Button>
+                <Button noOverflow title='Applies illustrative representation preset and Stylize it with cellShading' onClick={() => this.illustrative()} style={{ width: 'auto' }}>
+                    Illustrative
+                </Button>
+                <Button noOverflow title='Enable DOF and plastic material' onClick={() => this.stylized()} style={{ width: 'auto' }}>
+                    Stylized-DOF
+                </Button>
+            </div>
+            <ParameterControls params={style_params} values={this.plugin.managers.structure.component.state.options} onChangeValues={this.update} />
```

Comment:
Not sure about showing this, was hoping to keep the UI simple for this app. Could we settle on three quick styles for the user to pick from?

Reply:
Yes its fine, look at the last commit. I still exposed the renderer and trackball settings in the left panels. I think its important to have access to lighting and control settings.